assets/src/edit-story/app/rightClickMenu/provider.js

```diff
@@ -193,6 +278,88 @@ function RightClickMenuProvider({ children }) {
     [handleCopyPage, menuItemProps, handleDeletePage, handlePastePage]
   );
 
+  const foregroundMediaItems = useMemo(
+    () => [
+      ...defaultItems,
+      {
+        label: RIGHT_CLICK_MENU_LABELS.SEND_BACKWARD,
+        separator: 'top',
+        // TODO: this shortcut does not exist yet. Add shortcut to editor.
```

Comment:
made a ticket for this! https://github.com/google/web-stories-wp/issues/8440

Reply:
thank you!